maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/AbstractSurefireMojoToolchainsTest.java

```diff
@@ -106,39 +94,17 @@ public void shouldFallthroughToBuildContextWhenNoSpecSet() throws Exception {
         assertThat(actual).isSameAs(expectedFromContext);
     }
 
-    @Test
-    public void shouldReturnNoToolchainInMaven32() throws Exception {
-        Toolchain toolchain = invokeMethod(
-                AbstractSurefireMojo.class,
-                "getToolchainMaven33x",
-                MockToolchainManagerMaven32.class,
-                new MockToolchainManagerMaven32(null),
-                mock(MavenSession.class),
-                emptyMap());
-        assertNull(toolchain);
-    }
-
     @Test(expected = MojoFailureException.class)
-    public void shouldThrowMaven33xToolchain() throws Exception {
-        invokeMethod(
-                AbstractSurefireMojo.class,
-                "getToolchainMaven33x",
-                MockToolchainManager.class,
-                new MockToolchainManager(null, null),
-                mock(MavenSession.class),
-                emptyMap());
+    @Ignore
+    public void shouldThrowToolchain() throws Exception {
+        invokeMethod(AbstractSurefireMojo.class, "getToolchain");
     }
 
     @Test
-    public void shouldGetMaven33xToolchain() throws Exception {
+    @Ignore
+    public void shouldGetToolchain() throws Exception {
         Toolchain expected = mock(Toolchain.class);
-        Toolchain actual = invokeMethod(
-                AbstractSurefireMojo.class,
-                "getToolchainMaven33x",
-                MockToolchainManager.class,
-                new MockToolchainManager(expected, null),
-                mock(MavenSession.class),
-                emptyMap());
+        Toolchain actual = invokeMethod(AbstractSurefireMojo.class, "getToolchain");
```

Comment:
I have disabled a few tests because they do not work, but I believe that they are redundant now because we don't use any reflection. I am inclined to delete all disabled ones. WDYT?

Reply:
delete it